doc/jsk_perception/nodes/image_publisher.md

```diff
@@ -21,6 +21,13 @@ Publish image from loaded file.
 * `~frame_id` (str default: `camera`)
 * `~rate` (Float default: `1.0`)
 
+* `~fovx` (Float default: `None`)
```

Comment:
Please add unit of `~fovx` and `~fovy`. Maybe [degree]?

Reply:
Modified